.docker/ci/Dockerfile

```
@@ -1,23 +1,25 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
 MAINTAINER Dave Coleman dave@picknik.ai
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/root/ws_moveit
+ENV ROS_DISTRO crystal
+
+WORKDIR $ROS_WS
 
 # Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
-RUN \
-    mkdir src && \
+
+RUN mkdir src && \
     cd src && \
     #
     # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init --shallow . https://raw.githubusercontent.com/PickNikRobotics/moveit2/moveit2-ci/moveit.rosinstall && \
```

To be applied before merging:
```suggestion
    wstool init --shallow . https://raw.githubusercontent.com/PickNikRobotics/moveit2/${ROS_DISTRO}-devel/moveit.rosinstall && \
```